MiniApp/Classes/MiniAppSdkConfig.swift

```diff
@@ -0,0 +1,45 @@
+import Foundation
+
+public class MiniAppSdkConfig {
+    var baseUrl: String? {
+        didSet(value) {
+            if value?.count ?? 0 == 0 {
+                baseUrl = nil
+            }
+        }
+    }
+    var rasAppId: String? {
+        didSet(value) {
+            if value?.count ?? 0 == 0 {
+                rasAppId = nil
+            }
+        }
+    }
+    var subscriptionKey: String? {
+        didSet(value) {
+            if value?.count ?? 0 == 0 {
+                subscriptionKey = nil
+            }
+        }
+    }
+    var hostAppVersion: String? {
+        didSet(value) {
+            if value?.count ?? 0 == 0 {
+                hostAppVersion = nil
+            }
+        }
+    }
+    var key: String {
```

Comment:
On android, the `key` is being used to store different instances of the api client. But on iOS, there's only the single instance of the api client, so I think this `key` isn't being used for anything, right?